app/utilities/mailing.py

```diff
@@ -6,22 +6,48 @@ async def send_verification_mail(db_account):
     data = {
         'from': 'FastSurvey <noreply@admin.fastsurvey.io>',
```

Comment:
I would like to stick with `noreply@fastsurvey.io` -- less email addresses are easier to manage

Reply:
Will do!